loadpipeofficial/Hamza-medusa
# Gift 16 category

Changes: 
- added category in seed for giftcards
- added collection in seed for giftcards 
- added new store to sell giftcards (in setup) 
- modified setup script accordingly 

To test: 
- run setup-0
- run setup-1
- ensure that database contains a "Gift Cards" store, category, and collection
